app/models/spree/option_value_decorator.rb

```diff
@@ -1,10 +1,12 @@
 Spree::OptionValue.class_eval do
 
-  has_attached_file :image,
-    :styles        => { mini: '32x32>', normal: '128x128>' },
-    :default_style => SpreeVariantOptions::VariantConfig[:option_value_default_style],
-    :url           => SpreeVariantOptions::VariantConfig[:option_value_url],
-    :path          => SpreeVariantOptions::VariantConfig[:option_value_path]
+  has_attached_file(:image, {
+    styles: { mini: "32x32>", small: "64x64>", normal: "128x128>" },
+    default_style: SpreeVariantOptions::VariantConfig[:option_value_default_style],
+    url: SpreeVariantOptions::VariantConfig[:option_value_url],
+    path: SpreeVariantOptions::VariantConfig[:option_value_path]
+  })
```

Comment:
Indent the right brace the same as the first position after the preceding left parenthesis.